test/units/testsuite-13.sh

```diff
@@ -93,6 +93,29 @@ if echo test >> /run/host/os-release; then exit 1; fi
     fi
 }
 
+function check_machinectl_bind {
+    local _cmd='for i in $(seq 1 20); do if test -f /tmp/marker; then exit 0; fi; sleep 0.5; done; exit 1;'
```

Comment:
Not quite sure why, but the CentOS 7 job fails in `TEST-13-NSPAWN-SMOKE` with:

```
[    9.330946] systemd[1]: Started nspawn_machinectl_bind.service.
[    9.331115] systemd-nspawn[853]: + seq 1 20
[    9.331115] systemd-nspawn[853]: /bin/sh: seq: not found
[    9.331115] systemd-nspawn[853]: + exit 1
```

However, `seq` is in the [list of dependencies](https://github.com/systemd/systemd/blob/225d08b8790fb0927f9ab360e372f9f34bbddacb/test/test-functions#L94) and also seems to be installed into the VM image:

```
D: Installing /bin/rmdir
D: Installing /bin/sed
D: Installing /bin/seq
```
(https://jenkins-systemd.apps.ocp.ci.centos.org/job/upstream-centos7/1912/artifact//systemd-centos-ci/artifacts_Jp0Tvg/testsuite-logs-upstream.56s/setup-TEST-13-NSPAWN-SMOKE_PASS.log)

Reply:
ah I thought it was a built-in - the command is running inside nspawn which is a smaller busybox image, will add that too

Reply:
After a quick scroll through the test scripts I'd say that creating a busybox symlink for `seq` in a similar fashion to the already present ones _could theoretically_ help:

https://github.com/systemd/systemd/blob/c17b2155e53e8fae398a35019f039d7cd104e8c6/test/create-busybox-container#L26-L30

I wonder though how the test passed in other runs.

Reply:
> ah I thought it was a built-in - the command is running inside nspawn which is a smaller busybox image, will add that too

Beat me to it :-)

Reply:
yep, pushed - no idea how it works elsewhere and on my machines, there will be more commands leaking in somehow